core/src/main/scala/org/apache/spark/memory/MemoryManager.scala

```diff
@@ -255,7 +255,7 @@ private[spark] abstract class MemoryManager(
     }
     val size = ByteArrayMethods.nextPowerOf2(maxTungstenMemory / cores / safetyFactor)
     val default = math.min(maxPageSize, math.max(minPageSize, size))
-    conf.getSizeAsBytes("spark.buffer.pageSize", default)
+    conf.getSizeAsBytes(BUFFER_PAGESIZE.key, default)
```

Comment:
If this is a byteConf, can this just use `conf.get()`?

Reply:
@srowen 
Updated the code to use conf.get.
initially, I thought the same, But there is no method to provide the default value sparkConf class.
Now I am checking the values, if not present initializing with a default value.